### ISSUE TITLE
fuzzy fingerprinting: can match to a car while missing crucial ECUs

### ISSUE THREAD

**Issue by sshane** [commaai/openpilot#28483]
For example, if the radar or camera refuses to respond (or any other important ECU), but we get enough matches with other ECUs, we would match when we really shouldn't.

**Comment by sshane** [comment]
@jayvin100 I would recommend trying a different OBD-C cable or ethernet cable since this looks to be a hardware issue. panda drops some messages on both OBD 2 port and powertrain bus right around when we expect responses from the camera ECU.

![image](https://github.com/commaai/openpilot/assets/25857203/2de45791-b4fb-448c-a98c-41ab376fe6e3)


Nothing back from ECU:

![image](https://github.com/commaai/openpilot/assets/25857203/559c366c-15da-43ea-b935-9973d32445f0)

@robbederks does this look like a harness box issue or cable issue?

**Comment by robbederks** [comment]
Does this work on other 2022 civics? Strange that there would be a drop on two buses

**Comment by jayvin100** [comment]
> Just looking into that case since we would fail to fingerprint for that user after this change.
> 
> @jayvin100 I would recommend trying a different OBD-C cable or ethernet cable since this looks to be a hardware issue. panda drops some messages on both OBD 2 port and powertrain bus right around when we expect responses from the camera ECU.
> 
> `8fcafab4167b8c6c|2023-06-08--12-24-23`: ![image](https://user-images.githubusercontent.com/25857203/244853126-2de45791-b4fb-448c-a98c-41ab376fe6e3.png)
> 
> Nothing back from ECU:
> 
> ![image](https://user-images.githubusercontent.com/25857203/244853161-559c366c-15da-43ea-b935-9973d32445f0.png)
> 
> @robbederks does this look like a harness box issue or cable issue?

Battery just died, got a fresh one installed. Not sure if that could cause the issue also am on sunnypilot running nicki model, shall I install back stock openpilot to see if this issue persist?

**Comment by adeebshihadeh** [comment]
Yes, make sure you can repro on stock openpilot. Do a bunch of these users have a community built harness or something?

**Comment by jayvin100** [comment]
> Yes, make sure you can repro on stock openpilot. Do a bunch of these users have a community built harness or something?

Have been running master-ci since this issue was brought up last week. It's has been close to a week now, is it giving the expected responses? @sshane

**Comment by adeebshihadeh** [comment]
@sshane isn't this fixed?

**Comment by sshane** [comment]
this issue? no. the civic blocker? yes. i need to check some data to make sure this isn't the case for other cars and we can merge https://github.com/commaai/openpilot/pull/28481
